client/src/components/Overview/StyleSelector.jsx

```diff
@@ -1,9 +1,82 @@
 import React from 'react';
 
-const StyleSelector = () => {
+const StyleSelector = ({currentStyle, styles, changeCurrentStyle}) => {
+  // THIS WILL GET MOVED TO THE CSS FILE LATER
+  const styleSelectorThumbnail = {
+    width: '55px',
+    height: '55px',
+    borderRadius: '50%',
+  };
+
+  const checkmark = {
+    position: 'absolute',
+    top: '-5px',
+    right: '-5px',
+    padding: '5px',
+    borderRadius: '50%',
+    backgroundColor: 'green',
+    color: 'white',
+    width: '12px',
+    height: '12px',
+    display: 'flex',
+    alignItems: 'center',
+  };
+
+  const styleSelector = {
+    position: 'relative',
+    display: 'inline-block',
+  };
+
+  const onChange = (event) => {
+    let style;
+    for (let i = 0; i < styles.length; i++) {
+      if (styles[i].name === event.target.name) {
+        style = styles[i];
+        break;
+      }
```

Comment:
Is there a way to set the style without looping through styles? I'm thinking since there's already a loop in the thumbnails section, you might be able to either pass in styles[i] in the onClick listener itself or save some other metadata referencing styles[i] and reference that in your onChange handler.